src/sonic-ctrmgrd/ctrmgr/kube_commands.py

```diff
@@ -523,7 +530,7 @@ def _do_clean(feat, current_version, last_version):
         if last_version in version_dict:
             del version_dict[last_version]
 
-        versions = [item[DOCKER_ID] for item in version_dict.values()]
+        versions = [item[IMAGE_ID] for item in version_dict.values()]
```

Comment:
the "versions" here means image id list ?

Reply:
> the "versions" here means image id list ?

Yes, it's image id list which we need to remove. Let me rename the variable